tests/e2e/plugins/module-setup-tagmanager-no-account.php

```diff
@@ -20,14 +20,11 @@ function () {
 
 		register_rest_route(
 			REST_Routes::REST_ROOT,
-			'modules/tagmanager/data/accounts-containers',
+			'modules/tagmanager/data/accounts',
```

Comment:
Why was this changed here? Related though: I believe we should remove the `accounts-containers` datapoint entirely now since it's no longer used?

Reply:
We don't call the datapoint anymore so this was changed for the E2E test to continue to work. Alternatively we could add it as an additional endpoint to mock, but again the former is going to be removed and isn't currently used.